lib/src/chart/bar_chart/bar_chart_painter.dart

```diff
@@ -317,6 +326,9 @@ class BarChartPainter extends AxisChartPainter<BarChartData> {
               canvasWrapper.drawRRect(barRRect, _barPaint);
               canvasWrapper.restore();
             }
+            // draw border stroke for each stack item
+            _drawRodStacksBorderStroke(

```

Comment:
Above we iterated over stack items to draw them, why we iterated over them again inside this function?
I think we can use one iteration over stack items at all.

Reply:
You're right, I've changed the code.